csm/plugins/cortx/rgw.py

```diff
@@ -115,22 +111,19 @@ def _supress_response_keys(self, operation, response) -> Any:
             operation (str): IAM operation.
             response (dict): Response from s3 server.
 
-        Raises:
-            CsmInternalError
-
         Returns:
             Modified response.
         """
         suppressed_response = response
         keys = self._api_suppress_payload_schema.get(operation)
-        Log.debug(f"Suppressing {keys} keys from raw response.")
         if keys:
+            Log.info(f"Suppressing {keys} keys from raw response.")
             try:
                 for key in keys:
                     suppressed_response = Utility.remove_json_key(suppressed_response, key)
             except Exception as e:
                 Log.error(f"Error occured while suppressing {keys} keys from response: {e}")
-                raise CsmInternalError(const.S3_CLIENT_ERROR_MSG)
+                raise e
```

Comment:
same as above

Reply:
this is the private method and its error handling is taken care at calling method.